src/NzbDrone.Core/RootFolders/UnmappedFolder.cs

```diff
@@ -4,5 +4,6 @@ public class UnmappedFolder
     {
         public string Name { get; set; }
         public string Path { get; set; }
+        public string Id { get; set; }
```

Comment:
`Id` is reserved for the `Id` of the record, `TvdbId` would be more accurate since that assumption is being made in the client. It should also be an integer instead of a string. Probably a nullable int so we don't return 0 if it's unknown.